js/src/common/components/RequestErrorModal.js

```diff
@@ -7,17 +7,25 @@ export default class RequestErrorModal extends Modal {
 
   title() {
     return this.props.error.xhr
-      ? this.props.error.xhr.status+' '+this.props.error.xhr.statusText
+      ? `${this.props.error.xhr.status} ${this.props.error.xhr.statusText}`
       : '';
   }
 
   content() {
+    const { error, formattedError } = this.props;
+
     let responseText;
 
-    try {
-      responseText = JSON.stringify(JSON.parse(this.props.error.responseText), null, 2);
-    } catch (e) {
-      responseText = this.props.error.responseText;
+    if (!formattedError) {
```

Comment:
When there's an `else`, there's no need for a negated `if` condition - this avoids dealing with double negatives.

Reply:
True, makes sense. Will fix soon™.